### PR TITLE
Fix the problem of triple header losing information

### PR DESCRIPTION
**Pull request by EvenLjj**
Fix the problem of triple header losing information

### CODE REVIEW

File: remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapter.java [hunk 1]
```diff
@@ -81,6 +82,10 @@ public static void beforeSend(SofaRequest sofaRequest, ConsumerConfig consumerCo
         header.put(RemotingConstants.HEAD_METHOD_NAME, sofaRequest.getMethodName());
         header.put(RemotingConstants.HEAD_TARGET_SERVICE, sofaRequest.getTargetServiceUniqueName());
         header.put(RemotingConstants.HEAD_TARGET_APP, sofaRequest.getTargetAppName());
+        Map<String, Object> requestProps = sofaRequest.getRequestProps();
```
Comment (OrezzerO) [inline remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapter.java:85]: Consider the priority of each header key. For example,  `RemotingConstants.HEAD_TARGET_SERVICE` uses the value of `sofaRequest.getTargetServiceUniqueName()`, can it be replaced by `sofaRequest.getRequestProps()`?

File: remoting/remoting-triple/src/test/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapterTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+public class TripleTracerAdapterTest {
+
+    @Test
+    public void testBeforeSend() {
+        SofaRequest sofaRequest = new SofaRequest();
+        sofaRequest.addRequestProp("triple.header.key", "triple.header.value");
+        ConsumerConfig consumerConfig = new ConsumerConfig();
+        Metadata metadata = new Metadata();
+        TripleTracerAdapter.beforeSend(sofaRequest, consumerConfig, metadata);
+        Assert.assertTrue(metadata.containsKey(TripleHeadKeys.getKey("triple.header.key")));
```
Comment (OrezzerO) [inline remoting/remoting-triple/src/test/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapterTest.java:39]: Header value needs to be verified.

**Review by OrezzerO — DISMISSED** [review]

File: remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapter.java (hunk 1 above)
Comment (EvenLjj) [reply remoting/remoting-triple/src/main/java/com/alipay/sofa/rpc/tracer/sofatracer/TripleTracerAdapter.java:85]: There is indeed the problem of coverage. But I think this is reasonable. Users can customize the key to overwrite the original value here, and the original logic of bolt is the same. Of course, it is generally not recommended to use this method to overwrite.

**Review by chuailiwu — APPROVED** [review]

**Review by leslieSll — APPROVED** [review]
LGTM

**Pull request merged**